pkg/update/update_test.go

```diff
@@ -0,0 +1,29 @@
+package update
+
```

Comment:
Hi there! 

the test file relies on actual credentials and network connectivity which I would like to avoid for portability and speed of testing. It's a bad coding practice except for database testing.

Please either remove the test file and testify OR rework the test file to use a fake local API server to imitate your DNS provider that would accept any credentials and return the same deterministic response. I think testify has such unit testing ready http server built-in.

Reply:
I will delete the test file, the test file is just make sure the func is work,third party api is a dependence, I should make it is worked